RecoHGCal/GraphReco/python/peprCandidateFromHitProducer_cfi.py

```diff
@@ -0,0 +1,46 @@
+# coding: utf-8
+
+"""
+Initialization file for the WindowInference module.
+"""
+
+
+__all__ = ["peprCandidateFromHitProducer"]
+
+
+import math
+
+import FWCore.ParameterSet.Config as cms
+
+
+peprCandidateFromHitProducer = cms.EDProducer("peprCandidateFromHitProducer",
+    # the collections of rechits to use
+    recHitCollections=cms.VInputTag(
+        cms.InputTag("HGCalRecHit", "HGCEERecHits"),
+        cms.InputTag("HGCalRecHit", "HGCHEFRecHits"),
+        cms.InputTag("HGCalRecHit", "HGCHEBRecHits"),
+    ),
+    tracks = cms.InputTag("generalTracks"),
+    simClusters = cms.InputTag("mix", "MergedCaloTruth"),
+    # whether or not the model in the graph expects a batch dimension
+    batchedModel=cms.bool(True),
+    # graph to the trained model
+    #graphPath=cms.string("graph.pb"),
+    tritonPath=cms.string("/afs/cern.ch/work/g/gvonsem/public/HGCAL/ML/HGCalML/triton/"),
+    inpipeName=cms.string("arrayspipe"),
+    outpipeName=cms.string("arrayspipe_pred"),
```

Comment:
Best thing would be if the pipes are named from within the producer using its PID, to make sure two are not interfering on the same machine.
#include <sys/types.h>
#include <unistd.h>
pid_t pid = getpid(); // and then stringify this

Reply:
UPDATE:
With the latest changes, only the forward client is needed, and can be run from one script that has no other dependencies. Therefore, let's just copy that script to the GraphReco directory.

Reply:
The pipe names now have PID prepended. I have not yet copied the script to GraphReco, I just ran it from HGCalML/triton.